README.md

```diff
@@ -1301,18 +1301,26 @@ After receiving an `access_token` or `id_token`, add it to the `tokenManager` to
 ```javascript
 authClient.token.getWithPopup()
 .then(function(idToken) {
-  authClient.tokenManager.add('my_id_token', idToken);
+  authClient.tokenManager.add('idToken', idToken);
```

Comment:
nitpick: In the SIW docs, we use `.add('id_token')` - we should use one style everywhere. What's your preference?